### DISCUSSION TITLE
Aircrack-ng question still needs answer (From a Class with ZSec)

### DISCUSSION THREAD

**Discussion by ChillVibesMushroom** [aircrack-ng/aircrack-ng#2534] [Q&A]
I run these commands and see data flowing but I do not see any devices so then I find an AP with a device that shows one packet flowing and then the entire router disappears after a DOS attack and I don't know what to do 

sudo airmon-ng check kill

sudo airmon-ng start wlan0

sudo airodump-ng wlan0 --bssid --channel

sudo aireplay-ng -a bssid -c client --ignore-negative-one -0 3

*this is on a 5Ghz channel that showed one device and after I ran aireplay-ng the entire router dissappeared from the spectrum of frequencies being captured with airodump-ng  as in I can't see it at all anymore I even went to go back and check for channel switching and it didn't even show the essid

 Aircrack-ng 1.6

Realtek Semiconductor Co., Ltd. RTL8821CE 802.11ac PCIe Wireless Network Adapter

*This has even been happening at times on my alfa wireless adapter which is 2.4 Ghz

*I was trying to check for channel hopping of a sort but I need to figure out why this is happening because there may come a time when I'm in a future engagement and I need aircrack-ng to work correctly at one point I have failed at something that led to the loss of curcial information without giving too much detail 

Linux kali 5.18.0-kali5-amd64 #1 SMP PREEMPT_DYNAMIC Debian 5.18.5-1kali6 (2022-07-07) x86_64 GNU/Linux
  
![proof](https://user-images.githubusercontent.com/99580833/227712240-972055c4-e54f-4aeb-a744-b7cbbf81883f.png)
![wherediditgoHoss](https://user-images.githubusercontent.com/99580833/227712241-a99150b5-014e-433c-b2ca-6e2b01151d63.png)

**Comment by Mister-X-** [comment]
Such question should be asked to ZSec. They are not paying us for commercial support.
